packages/sampler/src/MoonstoneEnvironment/MoonstoneEnvironment.js

```diff
@@ -136,20 +145,24 @@ const StorybookDecorator = (story, config) => {
 
 	const DevelopmentConfig = {
 		defaultProps: {
-			'debug aria': false,
 			'debug layout': false,
 			'debug spotlight': false
 		},
 		groupId: 'Development'
 	};
 
+	const debugAriaOption = select('debug aria', debugAriaOptions, DevelopmentConfig, getPropFromURL('debug aria'));
+
 	const classes = {
-		aria: boolean('debug aria', DevelopmentConfig, (getPropFromURL('debug aria') === 'true')),
+		[`aria ${debugAriaOption}`]: debugAriaOption && true,
 		layout: boolean('debug layout', DevelopmentConfig, (getPropFromURL('debug layout') === 'true')),
 		spotlight: boolean('debug spotlight', DevelopmentConfig, (getPropFromURL('debug spotlight') === 'true'))
 	};
-	if (Object.keys(classes).length > 0) {
-		classes.debug = true;
+
+	for (const key in classes) {
+		if (typeof classes[key] !== 'undefined') {
```

Comment:
If we set a fallback value when calling `getPropFromURL`, we don't need to check whether its value is `undefined` or not.